fhir-persistence-cos/src/main/java/com/ibm/fhir/persistence/cos/payload/FHIRPayloadPersistenceCosImpl.java

```diff
@@ -115,20 +107,35 @@ private static String makeObjectName(int resourceTypeId, String logicalId, int v
      * @param payloadKey
      * @return
      */
-    private static String makeObjectName(PayloadKey payloadKey) {
-        return payloadKey.getPayloadId();
+    private static String makeObjectName(PayloadPersistenceResponse payloadKey) {
```

Comment:
should the param should be renamed as well?

Reply:
No longer used so I have removed